website/docs/r/cluster.html.markdown

```diff
@@ -17,7 +17,17 @@ description: |-
 ~> **IMPORTANT:**
 <br> &#8226; New Users: If you are not already using `mongodbatlas_cluster` for your deployment we recommend starting with the [`mongodbatlas_advanced_cluster`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/advanced_cluster).  `mongodbatlas_advanced_cluster` has all the same functionality as `mongodbatlas_cluster` but also supports multi-cloud clusters.  
 <br> &#8226; Free tier cluster creation (M0) is supported.
-<br> &#8226; Shared tier clusters (M0, M2, M5) cannot be upgraded to higher tiers via API or by this Provider.  WARNING! If you attempt to upgrade from an existing shared tier cluster that you manage with this Provider to a dedicated cluster (M10+) Terraform will see it as a request to destroy the shared tier cluster and as a request to create a dedicated tier cluster, i.e. Terraform will not see it as a request to upgrade.   If you accept the plan in this case the shared tier cluster would be destroyed and you would lose the data on that cluster. Do not attempt to upgrade from the shared to dedicated tier via this Provider, it is not supported!  
+<br> &#8226; Shared tier clusters (M0, M2, M5) are supported.
+```
+provider_instance_size_name = "M0"
+provider_name               = "TENANT"
+backing_provider_name       = "AWS"
+```
```

Comment:
We have examples in the examples below of both a free shared tier and regular shard tier so I don't think we need to state this.  I get why you added, due to the free tier explicitly being called out above in line 19. We just have that as it used to not be the case so we still get asked on it.  Hence leaving it specifically called out here. 
```suggestion
```